src/frontend/components/data-table/lib/component.js

```diff
@@ -8,7 +8,6 @@ import 'datatables.net-rowreorder-bs4'
 import { setupDisclosureWidgets, onDisclosureClick } from 'components/more-less/lib/disclosure-widgets'
 import { moreLess } from 'components/more-less/lib/more-less'
 import { bindToggleTableClickHandlers } from './toggle-table'
-import TypeaheadBuilder from 'util/typeahead'
```

Comment:
This isn't related (strictly speaking) but needs removing, and as I was working on the file :smiley: